doc/source/cluster/kubernetes/user-guides/aws-eks-gpu-cluster.md

```diff
@@ -35,31 +35,31 @@ Create a GPU node group for Ray GPU workers.
    * Disk size: 1024 GB
    * Desired size: 1, Min size: 0, Max size: 1
 
-> **Note:** If you encounter permission issues with `kubectl`, follow "Step 2: Configure your computer to communicate with your cluster"
-in the [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#).
+   > **Note:** If you encounter permission issues with `kubectl`, follow "Step 2: Configure your computer to communicate with your cluster"
```

Comment:
The "Note" section here serves as a supplement to point (2) which is the first step using `kubectl`. Perhaps, it would be better positioned slightly lower down.

* Screenshot
  <img width="805" alt="Screen Shot 2023-09-12 at 2 33 03 PM" src="https://github.com/ray-project/ray/assets/20109646/9fae9e38-f3e5-45bf-b8c4-1044dc6254e5">